From f2bb5543a537f644bd70b2190855593092f0cd82 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Wed, 15 Nov 2017 22:48:27 +0800
Subject: [PATCH 2804/5242] MLK-16821 ARM64: dts: freescale: imx8mq: increase
 CMA size to 1GB

commit  5e91aba19a88b615ae297d0f38a3c9ef09ee58a1 from
https://source.codeaurora.org/external/imx/linux-imx.git

Increase CMA size to 1GB to support dual 4K video playback.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 2486cdf..964abaf 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -80,7 +80,7 @@
 		linux,cma {
 			compatible = "shared-dma-pool";
 			reusable;
-			size = <0 0x28000000>;
+			size = <0 0x40000000>;
 			alloc-ranges = <0 0x40000000 0 0x70000000>;
 			linux,cma-default;
 		};
-- 
1.7.9.5

